### ISSUE TITLE
'SequentialTransform' object has no attribute 'points'

### ISSUE THREAD

**Issue by MengyunQ** [BioMedIA/deepali#138]
I have a code that used to work:
`def warp_transform_on_mesh(transform, target_mesh_file, warped_mesh_path):
    target_mesh = pv.read(target_mesh_file)
    target_points = torch.from_numpy(target_mesh.points).unsqueeze(0)
    target_points = target_points.to(device=transform.device)
    warped_target_points = transform.points(target_points, axes="grid")
    target_mesh.points = warped_target_points.squeeze(0).detach().cpu().numpy()
    target_mesh.save(warped_mesh_path)`
    
but now " warped_target_points = transform.points(target_points, axes="grid")" has the error of "'SequentialTransform' object has no attribute 'points'".

Do you have any idea what reason it could be?

**Comment by aschuh-hf** [comment]
Which version are you using?

Looking at the code, `SequentialTransform` should be a subclass of `SpatialTransform`, and the base class implements the `points()` method.

**Comment by MengyunQ** [comment]
I just checked two different environments. The old one works but the new one didn't.

**Comment by aschuh-hf** [comment]
What is the output of `python -c 'import deepali.core'; print(deepali.core.__version__)` for the old and new environment?

**Comment by MengyunQ** [comment]
the output for old and new version is 0.3.2, 0.6.1

**Comment by MengyunQ** [comment]
I changed the version and it works now. Thank you!

**Comment by aschuh-hf** [comment]
Which version did you change to? The newer version (0.6.1) or did you have to revert to an older version (0.3.2)?
